<ns0:test-suite xmlns:ns0="urn:model.allure.qatools.yandex.ru" start="1563538058619" stop="1563538058622">
  <name>pytest_allure.test_post</name>
  <labels/>
  <test-cases>
    <test-case start="1563538058619" status="failed" stop="1563538058621">
      <description>
    用例描述：这是用例描述，出现在什么位置
    </description>
      <name>test_case_01</name>
      <failure>
        <message>AssertionError: assert (1 + 1) == 3</message>
        <stack-trace>@allure.feature('test_module_01')
    @allure.severity('critical')
    #test_case_01 用例tittle
    def test_case_01():
        '''
        用例描述：这是用例描述，出现在什么位置
        '''
&gt;       assert 1 + 1 == 3
E       assert (1 + 1) == 3

test_post.py:18: AssertionError</stack-trace>
      </failure>
      <attachments/>
      <labels>
        <label name="feature" value="test_module_01"/>
        <label name="severity" value="critical"/>
        <label name="thread" value="28116-MainThread"/>
        <label name="host" value="DS-WS-170432"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1563538058621" status="passed" stop="1563538058622">
      <name>test_case_02</name>
      <attachments/>
      <labels>
        <label name="feature" value="test_module_02"/>
        <label name="severity" value="blocker"/>
        <label name="thread" value="28116-MainThread"/>
        <label name="host" value="DS-WS-170432"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
  </test-cases>
</ns0:test-suite>
